assets/blocks/course-outline/module-block/edit.js

```diff
@@ -115,12 +108,15 @@ export const EditModuleBlock = ( props ) => {
 		);
 	}
 
+	const bordered =
+		undefined !== borderedSelected ? borderedSelected : moduleBorder;
```

Comment:
About the `moduleBorder`, maybe we could rename it when destructuring, just to avoid mistakes with the local attribute. Something like `outlineAttributes: { collapsibleModules, moduleBorder: outlineBordered },`. WDYT?